ui/imports/shared/views/TransactionDetailView.qml

```diff
@@ -212,23 +212,23 @@ Item {
                     id: baseFee
                     maxWidth: parent.width
                     primaryText: qsTr("Base fee")
-                    secondaryText: root.transaction !== undefined && !!root.transaction ?  qsTr("%1 Gwei").arg(RootStore.hex2Gwei(root.transaction.baseGasFees)) : ""
+                    secondaryText: root.transaction !== undefined && !!root.transaction ?  qsTr("%1").arg(LocaleUtils.currencyAmountToLocaleString(root.transaction.baseGasFees)) : ""
```

Comment:
AFAIK this is the way to check for `undefined` `typeof root.transaction !== "undefined"`.